src/extension.ts

```diff
@@ -107,6 +108,13 @@ class ZxdbConfigurationProvider implements vscode.DebugConfigurationProvider {
       return null;
     }
 
+    if (config.process.length > this.maxFilterNameLength) {
+      config.process = config.process.substring(0, this.maxFilterNameLength);
+      log.warn(
```

Comment:
Will the warning be prompted to users or just to the log of the extension?

Reply:
A warning will be prompted to the user